LICENSE

```diff
@@ -0,0 +1,192 @@
+Copyright 2018 The Bow Authors
```

Comment:
For now, I think we can remove this file - unless we should update authors...years...